src/relentless/optimize/objective.py

```diff
@@ -246,24 +250,56 @@ class RelativeEntropy(ObjectiveFunction):
     simulations, so this :class:`ObjectiveFunction` does not return a value.
     However, the gradient of the relative entropy with respect to the design
     variables :math:`\mathbf{x}` is much easier to compute as ensemble averages.
-    Currently, the :class:`RelativeEntropy` objective function supports only
-    :class:`~relentless.potential.pair.PairPotential` interactions. These interactions
-    are characterized by :math:`g_{ij}(r)`, an :class:`~relentless.ensemble.RDF`
-    for each pair of interacting types :math:`(i,j)` in each
-    :class:`~relentless.ensemble.Ensemble`. The gradient of :math:`S_{\rm rel}` is then:
+
+    The gradient of the relative entropy can be computed through direct ensemble
+    averaging or through integration of the radial distribution function (rdf)
+    of the target and model ensembles. When ``target`` is a :class:`str` and
+    ``thermo`` is a :class:`~relentless.simulate.analyze.WriteTrajectory`
```

Comment:
We should try to add some suborganization on this. I think you can do it like:

```suggestion
    of the target and model ensembles.
    
    .. rubric:: Direct ensemble average
    
    When ``target`` is a :class:`str` and ``thermo`` is a
    :class:`~relentless.simulate.analyze.WriteTrajectory`
```